### PR TITLE
Feature setup multer cloudinary and images upload

### PR DESCRIPTION
**Pull request by sholabioku**
### What this PR does (required):
- Add setup for cloudinary and multer
- Create a route for image upload

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -21,47 +25,71 @@ const server = http.createServer(app);
...
+    res.status(200).json({
+      message: 'images uploaded successfully',
+      data: urls,
+    });
+  } else {
+    res.status(405).json({
+      err: `${req.method} method not allowed`,
+    });
+  }
+});
```
Comment (ezeilo-su) [inline server/app.js:71]: You can use a post method ( `app.post`) here so you don't need to be checking the type of request for returning a response.

**Review by ezeilo-su — COMMENTED** [review]
looks good. One minor comment, though. :+1:

File: server/utils/multer.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    cb(null, true);
+  } else {
+    cb({ message: 'Unsupported file format' }, false);
+  }
+};
+
+const upload = multer({
+  storage,
+  fileFilter,
+  limits: {
```
Comment (rajivtitus) [inline server/utils/multer.js:23]: 👍🏽

File: server/app.js [hunk 3]
```diff
@@ -21,47 +25,65 @@ const server = http.createServer(app);
...
+app.use(express.static(join(__dirname, 'public')));
 
 app.use((req, res, next) => {
   req.io = io;
   next();
 });
 
-app.use("/auth", authRouter);
-app.use("/users", userRouter);
+app.post('/upload-images', upload.array('image'), async (req, res) => {
```
Comment (rajivtitus) [inline server/app.js:49]: Can you please move this route to the routes folder & the implementation to the controllers folder :)

**Review by rajivtitus — APPROVED** [review]
Excellent work with this 🚀 
- Great PR description and title too

Please addres pending comment before merging!

**Comment by sholabioku** [comment]
@rajivtitus Feedback addressed before merging

**Pull request merged**